### PR TITLE
feat: bring back cli command get_app_keys

### LINKED ISSUE #1017: Errors in few of the @stacks/cli command.
**Bug Description and Scenario:**

I have recently configured my Gaia hub. As the next step, I want to configure my Gaia hub as the default app storage hub.

I tried using the Stacks-cli way following the [forum post](https://forum.stacks.org/t/tech-preview-using-your-own-gaia-hub-with-the-cli/6160) and [Stacks-cli docs](https://docs.stacks.co/references/stacks-cli). I hit the below roadblocks. Appreciate any leads on the same. Thank you so much.

 1) **get_app_keys** and **gaia_sethub** commands are failing with the below error, unable to make getfile/putfile command.

 ```
   $ export BACKUP_PHRASE=“<snip>”
    gaia@gaia-VirtualBox:~$ stx get_app_keys “$BACKUP_PHRASE” lockwise.id.blockstack https://blocksurvey.io
    Enter password:
    Error: Non-base58 character
    at Object.decode (/home/gaia/.nvm/versions/node/v16.1.0/lib/node_modules/@stacks/cli/node_modules/base-x/src/index.js:115:11)
    at Object.decode (/home/gaia/.nvm/versions/node/v16.1.0/lib/node_modules/@stacks/cli/node_modules/bs58check/base.js:39:25)
    at Object.fromBase58Check (/home/gaia/.nvm/versions/node/v16.1.0/lib/node_modules/@stacks/cli/node_modules/bitcoinjs-lib/src/address.js:11:29)
    at BlockstackNetwork.coerceAddress (/home/gaia/.nvm/versions/node/v16.1.0/lib/node_modules/@stacks/cli/node_modules/blockstack/src/network.ts:84:42)
    at /home/gaia/.nvm/versions/node/v16.1.0/lib/node_modules/@stacks/cli/node_modules/blockstack/src/network.ts:353:62
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    Non-base58 character
```

   2) **stx authenticator** is starting on port 8888. But not sure how to proceed, since blockstack browser is not active anymore. Any guidance on this is appreciated.

   ```
 stx authenticator “http://gaia.blsy.me:3000” “$BACKUP_PHRASE” “https://hub.blockstack.org” 8888
    Press Ctrl+C to exit
    Authentication server started on 8888

```
   3)  With the new keychain, couldn’t proceed further. As I couldn’t get app-keys without domain name and register subdomain command also fails with the same error ,Invalid command arguments.

    
```
stx get_app_keys “$BACKUP_PHRASE” 1PQ4NSLCJLbge95AXd8tWmY6TX8KfxduPM https://blocksurvey.io
    Invalid command arguments: Schema “#/items/1/pattern” failed validation (problem: “should match pattern “^([0-9a-z_.±]{3,37})$|^([0-9a-z_±]{1,37}).([0-9a-z_.±]{3,37})$|^ID-[123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz]{1,35}$””, cause: “{“pattern”:”^([0-9a-z_.±]{3,37})$|^([0-9a-z_±]{1,37}).([0-9a-z_.±]{3,37})$|^ID-[123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz]{1,35}$"}")
```

 4)   Also, **make_keychain** command generates 24 word seed phrase and stx encrypt_keychain fails for the newly created key, as it throws the below error.

`Invalid command arguments: Schema “#/items/1/pattern” failed validation (problem: “should match pattern “^([0-9a-z_.±]{3,37})$|^([0-9a-z_±]{1,37}).([0-9a-z_.±]{3,37})$|^ID-[123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz]{1,35}$””, cause: “{“pattern”:”^([0-9a-z_.±]{3,37})$|^([0-9a-z_±]{1,37}).([0-9a-z_.±]{3,37})$|^ID-[123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz]{1,35}$"}")
`

**Is the bug present in the Blockstack CLI, Gaia hub, Connect or Blockstack Browser?**
The bug is reported in Stacks CLI . @stacks/cli package was used. 

**Setup**:

 Installed cli using `npm install -g @stacks/cli` . Tried the cli in both Ubuntu and Windows. Faced the same error. 

**Additional context**
This was originally reported in Stacks Forum. Adding the link [here](https://forum.stacks.org/t/help-required-for-setting-up-gaia-hub-using-cli/12001).

### PR DESCRIPTION
**Pull request by friedger**
## Description
This PR replaces the old code for get_app_keys with the calls to @stacks/wallet-sdk to generated app private keys.

Users can now generate their app private keys as follows:
```
export BACKUP_PHRASE="one race buffalo dynamic icon drip width lake extra forest fee kit"
stx get_app_keys "$BACKUP_PHRASE" 1 https://my.cool.dapp
```

Issue #1017 mentioned errors one `get_app_keys`

## Type of Change
- [ ] New feature
- [x] Bug fix
- [x] API reference/documentation update
- [ ] Other

## Does this introduce a breaking change?
The name argument was replaced by index argument for simplicity

## Are documentation updates required?
Help section has been updated

## Checklist
- [x] Code is commented where needed
- [ ] Unit test coverage for new or modified code paths
- [x] `npm run test` passes
- [x] Tag 1 of @yknl or @zone117x for review

### CODE REVIEW

**Comment by kyranjamie** [comment]
Per #1037, you'll need to rebase to master, should fix the CI issues `git fetch && git rebase origin/master`

File: packages/cli/package.json [hunk 1]
```diff
@@ -95,6 +95,7 @@
     "@stacks/stacking": "^2.0.0-beta.1",
     "@stacks/storage": "^2.0.0-beta.1",
     "@stacks/transactions": "^2.0.0-beta.1",
+    "@stacks/wallet-sdk": "^1.0.0-wallet-sdk.4",
```
Comment (zone117x) [inline packages/cli/package.json]: Looks like this PR is blocked on https://github.com/blockstack/stacks.js/pull/897 ?

**Review by zone117x — APPROVED** [review]
LGTM thanks @friedger!

**Review by reedrosenbluth — APPROVED** [review]
LGTM!

**Comment by agraebe** [comment]
@friedger could you resolve the conflict so we can merge?

**Comment by reedrosenbluth** [comment]
@friedger can you resolve the conflicts here? I tried to do so via the github web interface, but it doesn't seem to have worked 😓

**Comment by zone117x** [comment]
Fixed various issues with the PR and merged. Thanks @friedger!

**Comment by friedger** [comment]
Thank you for finishing this PR!

**Pull request merged**